.github/workflows/deploy-serverless.yml

```diff
@@ -73,6 +73,13 @@ jobs:
         with:
           type: ALL
 
+      - name: Setup Google Keyfile
+        run: printenv GOOGLE_KEYFILE > keyfile.json
+        working-directory: project
+        if: env.CHECK_GOOGLE_KEYFILE != ''
+        env:
+          CHECK_GOOGLE_KEYFILE: ${{ secrets.GOOGLE_KEYFILE }}
```

Comment:
You can't use `secrets` in an `if` statement, so we copy the value to an `env` in order to tests if it exists.